chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -1248,7 +1251,7 @@ impl Handler<WithSpanContext<PeerManagerMessageRequest>> for PeerManagerActor {
         ctx: &mut Self::Context,
     ) -> Self::Result {
         let msg_type: &str = (&msg.msg).into();
-        let (_span, msg) = handler_trace_span!(target: "network", msg, msg_type);
+        let (_span, msg) = handler_debug_span!(target: "network", msg, msg_type);
```

Comment:
Is the intent behind this change is to allow usable tracing with a debug level only?

Reply:
Yes, to allow tracing across nodes at both debug and trace levels.